src/servant/tt_001/ghost.h

```diff
@@ -0,0 +1,67 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+#include <servant.h>
+
+s16 D_us_801735B0, D_us_801735B0_;
```

Comment:
What's the reasoning for putting the bss and g_ServantDesc in the header?

Reply:
I thought it might be a good idea to move the declarations and function prototypes that were ghost specific into it's own header.  I don't have a solid reason for doing so other than that I thought that was how C projects typically are organized.  I can revert that.

Reply:
I reverted moving those to a header and also found the padding macro.